src/Core/Cache/ManifestCacheFactory.php

```diff
@@ -6,12 +6,15 @@
 use Monolog\Handler\ErrorLogHandler;
 use Monolog\Handler\StreamHandler;
 use Monolog\Logger;
-use Psr\Log\LoggerAwareInterface;
+use Psr\Cache\CacheItemPoolInterface;
 use Psr\Log\LoggerInterface;
 use Psr\SimpleCache\CacheInterface;
```

Comment:
```suggestion
use Psr\Cache\CacheItemPoolInterface as Psr6CacheInterface;
use Psr\Log\LoggerInterface;
use Psr\SimpleCache\CacheInterface as Psr16CacheInterface;
```
Same suggestion as with `DefaultCacheFactory` - for our purposes this is more informative than the actual interface names.

Reply:
As noted above, I'd prefer not to do this